### ISSUE TITLE
Unknown field (extra) on table roles

### ISSUE THREAD

**Issue by tiagosimoesdev** [glorand/laravel-model-settings#58]
Hi , 
duplicate of #43 , #31 . 

It worked before , I don't think i have updated any packages or made any changes .

Getting  `Unknown field (extra) on table roles` 

I tried 
`php artisan cache:clear`
`composer update glorand/laravel-model-settings` to 3.6.5 .

My model : 
```

<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Silber\Bouncer\Database\Concerns\IsRole;
use Glorand\Model\Settings\Traits\HasSettingsField;

class Role extends Model
{
    use IsRole, HasSettingsField;

    protected $connection = 'geral';



    protected $guarded = [];

    public $settingsFieldName = 'extra';
    public $defaultSettings = [
        'app' => 'false',
        'touch' =>  'false'
    ];
}
```

https://flareapp.io/share/x7KVQZPR#F45

**Comment by felixsc** [comment]
Just had the same issue. Had a previous model that worked fine and when I added another one it broke with the same error message.

The problem seems to be the caching in the hasSettingsField method of the HasSettingsField trait, because clearing the cache solved the problem for me.

### TLDR solution:

`php artisan cache:clear`
